src/ui-shell/header/header-item.component.ts

```diff
@@ -47,6 +52,15 @@ export class HeaderItem {
 	 */
 	@Input() routeExtras: any;
 
+	@Input() content: string;
+
+	/**
+	 * Set breakpoint to respond to the size of the window innerWidth. ie. the window
+	 * innerWidth to send the header item to the sidenav. Currently not used as styles
+	 * direct the response to the innerWidth of the window.
+	 */
+	@Input() breakpoint: number;
```

Comment:
Since this is unused we can just remove it ... We'd likely want a more comprehensive solution for programmatic control and use of the carbon breakpoints anyway